### ISSUE TITLE
The same training set, test set and training configuration, but different test results

### ISSUE THREAD

**Issue by wnynotfaker** [facebookresearch/detectron2#1825]
Hello,I used the rotating box to train and test twice with the same training set, test set and training configuration, and the test results differed by 3 to 4 percentage points for each indicator. Is this normal? I have used more online data augmentation.The way of data augmentation is somewhat randoms. Is it related to this?

Looking forward to your reply

**Comment by ppwwyyxx** [comment]
As the [issue template](https://github.com/facebookresearch/detectron2/blob/master/.github/ISSUE_TEMPLATE/unexpected-problems-bugs.md) mentions:

> If you expect the model to converge / work better, note that we do not give suggestions on how to train a new model. Only in one of the two conditions we will help with it: (1) You're unable to reproduce the results in detectron2 model zoo. (2) It indicates a detectron2 bug.

We provide configs & models with standard academic settings and expect users to have the knowledge to choose or design appropriate models & parameters for their own tasks.
